### PR TITLE
Self serve signed out help

### PR DESCRIPTION
**Pull request by akssinhaamit**
[Trello](https://trello.com/c/aaQBtrNL/505-s30-user-self-serve-signed-out-help-information-about-the-ons)

### CODE REVIEW

File: frontstage/views/help.py [hunk 1]
```diff
@@ -1,15 +1,57 @@
...
+
+@help_bp.route('/', methods=['GET'])
+def help_get():
+    return render_template('help/help.html', form=HelpForm())
+
+
+@help_bp.route('/', methods=['POST'])
+def help_submit():
+    form = HelpForm(request.values)
+    if request.method == 'POST' and form.validate():
```
Comment (warrenbailey) [inline frontstage/views/help.py:38]: is this check for post method necessary if the method only accepts post methods?

File: frontstage/views/help.py [hunk 2]
```diff
@@ -1,15 +1,57 @@
...
+
+@help_bp.route('/info-ons', methods=['GET'])
+def info_ons_get():
+    return render_template('help/help-info-ons.html', form=HelpInfoOnsForm())
+
+
+@help_bp.route('/info-ons', methods=['POST'])
+def info_ons_submit():
+    form = HelpInfoOnsForm(request.values)
+    if request.method == 'POST' and form.validate():
```
Comment (warrenbailey) [inline frontstage/views/help.py:53]: as above

**Review by warrenbailey — CHANGES_REQUESTED** [review]
Minor comments, looks fine to me

File: frontstage/models.py [hunk 3]
```diff
@@ -299,3 +299,29 @@ class AccountSurveyShareRecipientEmailForm(FlaskForm):
...
+
+class HelpForm(FlaskForm):
+    option = RadioField('Label', choices=[
+        ('value', 'info-ons'),
+        ('value', 'password'),
+        ('value', 'something-else')
+    ])
+
+    def validate(self):
+        if self.data['option'] is None:
```
[the remark this answers is not in the document]
Comment (akssinhaamit) [reply frontstage/models.py:312]: 👍

File: frontstage/models.py [hunk 4]
```diff
@@ -299,3 +299,29 @@ class AccountSurveyShareRecipientEmailForm(FlaskForm):
...
+
+
+class HelpInfoOnsForm(FlaskForm):
+    option = RadioField('Label', choices=[
+        ('value', 'ons'),
+        ('value', 'data'),
+        ('value', 'info-something-else')
+    ])
+
+    def validate(self):
```
[the remark this answers is not in the document]
Comment (akssinhaamit) [reply frontstage/models.py:324]: yes it has been there in many places.

File: frontstage/views/help.py (hunk 1 above)
Comment (akssinhaamit) [reply frontstage/views/help.py:38]: yups not required .. will take it out

File: frontstage/views/help.py (hunk 2 above)
Comment (akssinhaamit) [reply frontstage/views/help.py:53]: yups not required .. will take it out

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**